GitHub/Github_Add_New_Github_Issues_As_Page_In_Notion_Database.ipynb

```diff
@@ -189,13 +606,35 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "id": "cc5aca0f",
-   "metadata": {},
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-04-11T18:41:45.276824Z",
+     "iopub.status.busy": "2022-04-11T18:41:45.274326Z",
+     "iopub.status.idle": "2022-04-11T18:41:45.291597Z",
+     "shell.execute_reply": "2022-04-11T18:41:45.290989Z",
+     "shell.execute_reply.started": "2022-04-11T18:41:45.276783Z"
+    },
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "def notion_page_from_gh_issue(issue, notion_database_id):\n",
-    "\t...\n"
+    "    \n",
+    "    if len(df_notion) == 0:\n",
+    "        pass\n",
+    "    elif len(df_notion) != 0 and str(issue.issue_id) in df_notion.Issue_id.to_list():\n",
+    "        return \"issue already exists in database\"\n",
+    "    \n",
+    "    page = notion_client.page.create(database_id=notion_database_id, title = issue.issue_title)\n",
+    "    page.rich_text(\"URL\",issue.link_to_the_issue)\n",
```

Comment:
Over here I have added the URL as a rich text property because I had some issues while adding it as a URL property.
Can you let me know regarding this if anything can be done @Dr0p42
Thanks